examples/widgets/server.py

```diff
@@ -52,6 +61,11 @@ def chat_with_bot(input: Dict[str, Any]) -> Dict[str, Any]:
     }
 
 
+def chat_message_bot(input: Dict[str, Any]) -> BaseMessage:
```

Comment:
Are we able to make this also work with List[BaseMessage] to support a use case when we're working with an agent that can also invoke tools?